.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -346,7 +347,6 @@ jobs:
           env | grep '^GITHUB' > "/tmp/github_env_${GITHUB_RUN_ID}"
       - name: Test PyTorch
         env:
-          BUILD_ENVIRONMENT: !{{ build_environment }}-${{ matrix.config }}
```

Comment:
Removing this, context here: https://github.com/pytorch/pytorch/pull/64204#pullrequestreview-741980628